recipes/creme/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "creme" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/creme-ml/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "109e308070e023825efe0b3eb6f94898e59a24ce52da259828b2eb473dfed4c6"
+
+build:
+  skip: True  # [win or py<35]
```

Can this be `noarch`? It's fine if Windows then fails due to missing dependencies as the feedstock will only build for Linux.
```suggestion
  noarch: python
```
The python version constraint can be added using `python >=3.6` in both `host` and `run`.